src/components/edit/Interactions.jsx

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-virtualized-select';
 import autobind from 'react-autobind';
+import { remove } from 'lodash';
```

Comment:
if you `import remove from 'lodash/remove'` it'll keep the bundle size smaller (I think.)